### PR TITLE
Follow Kaltura plugins standrads

### PR DESCRIPTION
**Pull request by tan-tan-kanarek**

### CODE REVIEW

**Comment by tan-tan-kanarek** [comment]
@MosheMaorKaltura, please review.

File: plugins/capture_space/services/CaptureSpaceService.php [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+		if(kCaptureSpaceVersionManager::isLatest($os, $version)){
+			throw new KalturaAPIException(CaptureSpaceErrors::ALREADY_LATEST_VERSION, $version, $os);
+		}
+		
+		$hashValue = kCaptureSpaceVersionManager::getUpdateHash($os, $version);
+		if (!$hashValue) {
+			throw new KalturaAPIException(CaptureSpaceErrors::NO_UPDATE_IS_AVAILABLE, $version, $os);
+		}
+			
+		$path = "/api_v3/service/collaaj_collaajini/action/serve/serveUpdateAction/os/$os/version/$version";
```
Comment (gonenradai) [inline plugins/capture_space/services/CaptureSpaceService.php]: seems like the wrong path.
service name != this class' name

File: plugins/capture_space/services/CaptureSpaceService.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+	 * @action clientUpdates
+	 * @param string $os
+	 * @param string $version
+	 * @return KalturaCaptureSpaceUpdateResponse
+	 * @throws CaptureSpaceErrors::ALREADY_LATEST_VERSION
+	 * @throws CaptureSpaceErrors::NO_UPDATE_IS_AVAILABLE
+	 */
+	function clientUpdatesAction ($os, $version)
+	{
+		if(kCaptureSpaceVersionManager::isLatest($os, $version)){
```
Comment (gonenradai) [inline plugins/capture_space/services/CaptureSpaceService.php]: I can't find a file with name such as (k)CaptureSpaceVersionManager  in this branch,
am I missing something?

Comment (tan-tan-kanarek) [inline plugins/capture_space/services/CaptureSpaceService.php]: Added.

**Pull request merged**